scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
Does that mean we need to build binaries before docker build here in this script? And take out docker build from release script!?